src/api.js

```diff
@@ -114,8 +114,13 @@ async function getPoolTransaction (transactionId, axiosConfig = {}) {
  * @param {Object} transaction - Transaction data returned by TxUtils.generateL2Transaction
  * @returns {String} Transaction id
  */
-async function postPoolTransaction (transaction, axiosConfig = {}) {
-  return axios.post(`${baseApiUrl}/transactions-pool`, transaction, axiosConfig)
+async function postPoolTransaction (transaction, nextForgerUrls = [], axiosConfig = {}) {
+  if (!nextForgerUrls.includes(baseApiUrl)) {
+    nextForgerUrls.push(baseApiUrl)
+  }
+  return Promise.all(nextForgerUrls.map((apiUrl) => {
```

Comment:
What should the behavior here? I mean, `Promise.all` only succeeds if all of the requests succeed. What happens when one of them succeeds and the other one fails? Should we consider this case a failure or a success?

Reply:
You're right. I think it can fail for now, and we can merge this as is. But I'll ask Jordi tomorrow in case we need to change it